pkgs/by-name/mi/midisheetmusic/package.nix

```diff
@@ -74,7 +99,7 @@ stdenv.mkDerivation {
     mainProgram = "midisheetmusic.mono.exe";
     homepage = "http://midisheetmusic.com";
     license = licenses.gpl2;
-    maintainers = [ ];
+    maintainers = [ maintainers.mdarocha ];
     platforms = platforms.linux;
   };
```

Comment:
can you also remove `with lib;` statement? see #208242

Reply:
fixed